Library/Homebrew/cask/lib/hbc/dsl.rb

```diff
@@ -206,16 +204,6 @@ def sha256(arg = nil)
       @sha256 ||= arg
     end
 
-    def license(arg = nil)
```

Comment:
@jawshooah Fixed the IntegrationCommandTestCask failure, now here is File leak detected. 🤔 Trying dig into it.